examples/proc_FID_nutation.py

```diff
@@ -44,31 +44,31 @@
     s /= prscr.zeroth_order_ph(s["t2":0])
     s.ft("t2")
     fl.image(s, ax=ax1)
-    d_raw = s.C
     ax1.set_title("Signal pathway / ph0")
     # }}}
     # {{{ Look at phase variation
-    phase_var_s = s.C
-    d_uncorrected = phase_var_s.C.real.integrate("t2")
+    d_uncorrected = s.C.real.integrate("t2")
+    # PR again, you need to rename these
     for j in range(len(s.getaxis("beta"))):
         ph0 = prscr.zeroth_order_ph(phase_var_s["beta", j])
         phase_var_s["beta", j] /= ph0
     d_ind_ph0 = phase_var_s.real.integrate("t2")
     mysign = (d_ind_ph0 / d_uncorrected).angle / np.pi
     mysign = np.exp(1j * np.pi * mysign.run(np.round))
-    d_raw *= mysign
+    s *= mysign
     fl.image(d_raw, ax=ax2)
     ax2.set_title("Check phase variation along indirect")
     # }}}
+    # {{{ generate the table of integrals and fit
     s = s.real.integrate("t2")
     s.set_error(None)
     A, R, beta_ninety, beta = sp.symbols("A R beta_ninety beta", real=True)
     fl.plot(s, "o", ax=ax3, human_units=False)
-    f = psd.lmfitdata(s)
-    f.functional_form = (
+    s = psd.lmfitdata(s)
+    s.functional_form = (
```

Comment:
I didn't comment on this, but it's the same story.  The data changes as it is processed.  There is no need to spin up multiple copies of the data.  This is what we set out as one of the fundamental principles of clean code.